cn_indian_payroll/cn_indian_payroll/doctype/salary_structure_assignment_configuration/salary_structure_assignment_configuration.py

```
@@ -0,0 +1,9 @@
+# Copyright (c) 2025, Hybrowlabs technologies and contributors
```

Ruff (0.8.2)

The file is executable but no shebang is present

(EXE002)